resize-font

```diff
@@ -124,7 +124,7 @@ sub set_font {
 
 sub update_font_size {
   my ($self, $font, $delta) = @_;
-  my $regex = qr"(?<=size=)(\d+)";
+  my $regex = qr"(?<=size=)(\d+(\.\d+)?)";
```

Comment:
Since we don't use just the decimal value (we use the whole font size value) there's no need to capture it.
Can you change it to a non capturing group? Let me know if you need any help with it!

Reply:
Did it via commit --amend and force pushing. Hope that's not unorthodox.